src/components/ChecCard.vue

```diff
@@ -37,6 +37,8 @@ export default {
      * A "compact" variant that reduces border width and radius
      */
     compact: Boolean,
+    hoverable: Boolean,
+    active: Boolean,
```

Comment:
Want to add some doc-blocks for this?

Reply:
The linter was throwing https://eslint.org/docs/rules/no-multi-str before though. 

Yes, adding doc-blocks.